modules/swagger-codegen/src/main/java/com/wordnik/swagger/codegen/languages/RetrofitClientCodegen.java

```diff
@@ -0,0 +1,190 @@
+package com.wordnik.swagger.codegen.languages;
+
+import com.wordnik.swagger.codegen.*;
+import com.wordnik.swagger.models.Operation;
+import com.wordnik.swagger.models.properties.*;
+
+import java.util.*;
+import java.io.File;
+
+public class RetrofitClientCodegen extends DefaultCodegen implements CodegenConfig {
```

Comment:
Well, I took AndroidClientCodegen as reference, and it doesn't derive from JavaClientCodegen. Default constructor adds lot of stuff to additionalProperties and supportingFiles, so any change in java codegen can break its subclasses. Yes, I can just clear these lists before setting my own values, but it can break some stuff tat will be done in future versions of DefaultCodegen